tests/test_group_scores.py

```diff
@@ -75,11 +75,19 @@ def test_group_cm():
     "metric",
     # fmt: off
     [
-        "tpr", "fnr", "tnr", "fpr",
-        "topr", "tonr",
-        "tar", "frr", "trr", "far",
-        "acceptance_rate", "rejection_rate",
-    ]
+        "tpr",
+        "fnr",
+        "tnr",
+        "fpr",
+        "topr",
+        "tonr",
+        "tar",
+        "frr",
+        "trr",
+        "far",
+        "acceptance_rate",
+        "rejection_rate",
+    ],
     # fmr: on
```

Comment:
This is a typo and should be `# fmt: off`. It is possible that `ruff` ignores it, because it is within an [expression](https://docs.astral.sh/ruff/formatter/#format-suppression), in which case, we should remove the comments altogether.

Reply:
Indeed ruff ignores the comments, so I deleted them.